packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: 0.25;
```

Comment:
I totally agree with you. was just noting that removing the flex and just adding padding does not just work, because the button inherits the flex size from the inputs styling.

Might have to set the `flex: none` to override.